File: frontend/whole_app/lib/whole_app.dart
import 'package:flutter/material.dart';
import 'package:adder/adder.dart';
import 'package:word_counter/word_counter.dart';

class WholeApp extends StatelessWidget {
  @override
  Widget build(BuildContext context) {
    return DefaultTabController(
      length: 2,
      child: Scaffold(
        appBar: AppBar(
          bottom: TabBar(
            tabs: <Widget>[
              Tab(text: 'Adder'),
              Tab(text: 'Word Counter'),
            ],
          ),
        ),
        body: TabBarView(
          children: <Widget>[
            Adder(),
            WordCounter(),
          ],
        ),
      ),
    );
  }
}

File: backend/whole_app/src/main/java/wholeapp/WholeApp.java
package wholeapp;

import adder.Adder;
import org.jooby.Jooby;
import wordcounter.WordCounter;

public class WholeApp extends Jooby {

  {
    path("/", () -> {
      use(new Adder());
      use(new WordCounter());
    });
  }

  public static void main(final String[] args) {
    run(WholeApp::new, args);
  }

}
